msg/WorldBody.msg

```diff
@@ -0,0 +1,45 @@
+# Representation of a single body into the world 
+
+# Possible constants to fill into the type field
+uint8 UNDEFINED=0
+uint8 PRIMITIVE_BODY=1
+uint8 MESH_BODY=2
+uint8 URDF_BODY=3
+
+# The type of body that is represented
+uint8 type
+
+# The name with which the body shall be addressed in the world
+# Note: needs to be unique. If the name is already taken, an error will be returned
+string name
+
+# Pose of the body in the world. TF will be used to resolve the header information.
+geometry_msgs/PoseStamped pose
+
+# Flag that specifies whether this body is rigidly attached to an existing link
+bool rigidly_attached
+# Only used if rigidly_attached==True, otherwise ignored.
+# Name of the body that holds the link to which this body shall be connected.
+string parent_body_name
+# Name of the link to which this body shall be connected.
```

Comment:
We also need child_link_name, at least if it has more than 1. You could also add a comment that if a body only has 1 link we default to it in case the link_name is empty or invalid.